training/run_benchmarks/run.py

```diff
@@ -384,12 +459,15 @@ def prepare_containers_env_cluster(dp_path, case_log_dir, container_name,
     return True
 
 
-def clean_containers_env_cluster(dp_path, container_name, nnodes):
+def clean_containers_env_cluster(dp_path, container_name, nnodes, env_flag):
     '''Clean containers environments in the cluster. It will stop containers,
        and stop monitors.'''
-    RUN_LOGGER.info("a) Stop containers......")
-    stop_container_in_cluster(dp_path, container_name, nnodes)
-    RUN_LOGGER.info("b) Stop monitors......")
+    if env_flag == "baremetal":
+        RUN_LOGGER.info("a) Stop containers......")
+        stop_container_in_cluster(dp_path, container_name, nnodes)
+        RUN_LOGGER.info("b) Stop monitors......")
+    if env_flag == "container":
+        RUN_LOGGER.info("a) Stop monitors......")
```

Comment:
we can assume Ops has direct and explicit method to monitor, flagperf do not need to start any monitor when "container" mode